packages/core/src/base.css

```diff
@@ -1,4 +1,12 @@
-@import url("normalize.css");
 @import url("hds-design-tokens");
 @import url("utils/animations.css");
 @import url("utils/helpers.css");
+
+/*
+ * Normalize.css rule
+ * 1. Prevent adjustments of font size after orientation changes in iOS.
+ */
+
+html {
+  -webkit-text-size-adjust: 100%; /* 1 */
```

Comment:
Maybe some cleaning: remove the "`1.`" and "`/* 1 */`", because there is only one rule